nvflare/private/fed/server/sys_cmd.py

```diff
@@ -154,3 +163,15 @@ def report_resources(self, conn: Connection, args: List[str]):
         table = conn.append_table(["Sites", "Resources"])
         for k, v in site_resources.items():
             table.add_row([str(k), str(v)])
+
+    def report_env(self, conn: Connection, args: List[str]):
+        message = new_message(conn, topic=SysCommandTopic.REPORT_ENV, body="", require_authz=True)
```

Comment:
are we using the arg "client-name" if specified?

Reply:
Not needed here. The self.send_request_to_clients(conn, message) call knows what clients to send the message to, based on client-name arg(s). BTW, this is a common util that is used in many commands.